usr/share/rear/prep/NBKDC/default/40_prep_nbkdc.sh

```diff
@@ -0,0 +1,93 @@
+#
+# prepare stuff for NovaBACKUP DataCenter
+#
+
+# detect where and if DataCenter client is installed on the system running
+# REAR and include the needed bits in the recovery ISO
+
+if [ ! -f "$NBKDC_DIR/conf/client.properties" ]; then
+    NBKDC_DIR=$(readlink -f /proc/$(pgrep -nx rcmd-executor)/exe | sed 's/\/rcmd-executor*//g')
+    if  [ ! -f "$NBKDC_DIR/conf/client.properties" ] ; then
+        echo "Cannot find running NovaBACKUP DataCenter Agent!!"
```

Comment:
Please avoid using echo - use LogPrint (output goes to log and screen), Log instead